appveyor.yml

```diff
@@ -0,0 +1,17 @@
+version: 1.0.{build}
+environment:
+  nodejs_version: "4.2.2"
+os:
+  - Visual Studio 2015
+  - Visual Studio 2013
+install:
+  - ps: Install-Product node $env:nodejs_version x64
+  - npm install -g node-gyp
+  - npm install -g mocha
+  - curl -fLsS -o 'gtk.zip' 'http://ftp.gnome.org/pub/GNOME/binaries/win64/gtk+/2.22/gtk+-bundle_2.22.1-20101229_win64.zip'
+  - 7z x gtk.zip -oGTK > nul
+  - mv GTK/ C:/
+  - npm install
+build: off
+test_script:
+- cmd: mocha test/*.test.js
```

Comment:
Ahh, hmm, I would kind of like to get rid of the `pre*` as well. I don't think it's the right place to have that responsibility here.

Although, at least on unix, `node-gyp build` isn't the same as **`re`**`build`. Mine just outputs "nothing to be done"...